hack/test_k8s_e2e.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#check whether KUBECONFIG variable is set or not
+set -e
+# set environment variable and Kubeconfig
+KUBECONFIG=${KUBECONFIG:-""}
+LOCATION=${LOCATION:-""}
+
+if [ -z "$LOCATION" ]
+then
+echo "Location is not set"
+exit 1
```

Comment:
nit: indent this two line2 with extra 4 spaces

Reply:
ok sir